scripts/github-actions-setup

```diff
@@ -106,17 +106,29 @@ install_cni_plugins() {
 }
 
 install_runc() {
-    URL=https://github.com/opencontainers/runc/releases/download/"${VERSIONS["runc"]}"
-    BINARY=/usr/sbin/runc
-    sudo wget -O "$BINARY" "$URL/runc.$GOARCH"
-    sudo chmod +x "$BINARY"
-
-    # Verify the SHA256
-    SUMFILE=runc.sha256sum
-    wget "$URL"/$SUMFILE
-    grep -qw "$(sha256sum "$BINARY" | awk '{ print $1 }')" $SUMFILE
-    rm $SUMFILE
-
+    # Once https://github.com/opencontainers/runc/issues/4233 is fixed,
+    # we can delete go 1.21.9 and link the latest go toolchain.
+    GO_BINARY_URL="https://storage.googleapis.com/golang/go1.21.9.linux-$GOARCH.tar.gz"
+    GO_INSTALLATION_DIR="$(pwd)/go1.21.9"
+    RUNC_DIR="$(pwd)/runc"
+
+    mkdir -p "$GO_INSTALLATION_DIR"
+    curl -L -o - "$GO_BINARY_URL" | tar -xz -C "$GO_INSTALLATION_DIR"
+
+    # Clone the runc repository
```

Comment:
I am unsure if this comment adds any value, given that the line immediately below it is a Git clone of a repository.

Reply:
ok, I will remove all the redundant information.